src/views/transactions/components/selected-account/selected-account.view.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { TxType } from "@hermeznetwork/hermezjs/src/enums";
+
+import useSelectedAccountStyles from "src/views/transactions/components/selected-account/selected-account.styles";
+import { Account, EthereumAccount, FiatExchangeRates } from "src/domain/hermez";
+import FiatAmount from "src/views/shared/fiat-amount/fiat-amount.view";
+import { getFixedTokenAmount, getTokenAmountInPreferredCurrency } from "src/utils/currencies";
+import { AsyncTask, isAsyncTaskCompleted } from "src/utils/types";
+
+interface CommonSelectedAccountProps {
+  preferredCurrency: string;
+  showInFiat: boolean;
+  fiatExchangeRatesTask: AsyncTask<FiatExchangeRates, string>;
+}
+
+interface SelectedAccountDepositProps {
+  txType: TxType.Deposit;
+  account: EthereumAccount;
+  onClick: () => void;
+}
+
+interface SelectedAccountExitProps {
```

Comment:
Probably worth commenting why this one doesn't have an onClick. Not obvious that Exits are only accessible from account details

Reply:
done!